tests/pthread/Makefile

```diff
@@ -3,6 +3,8 @@ include ../Makefile.tests_common
 
 BOARD_BLACKLIST := arduino-mega2560 waspmote-pro
 # arduino-mega2560: unknown type name: clockid_t
+BOARD_BLACKLIST += arduino-uno arduino-duemilanove
```

Comment:
`arduino-uno` and `arduino-duemilanove` boards are blacklisted for the same reason (unknown type clockid_t) so no need to add those 2 extra lines, you can just append them to the first line and update the comment above.
There are a other places were this should be changed.